src/app/shared/shared.module.ts

```diff
@@ -112,12 +100,7 @@ export function apiConfigProvider(configService: ConfigurationService, appStateS
     TranslateModule
   ],
   //this is not elegant, for some reason the injection token from primeng does not work across federated module
-  providers: [
-    ConfirmationService,
-    LabelResolver,
-    { provide: DialogService, useClass: PortalDialogService },
-    { provide: Configuration, useFactory: apiConfigProvider, deps: [ConfigurationService, AppStateService] }
-  ],
+  providers: [ConfirmationService, LabelResolver, { provide: DialogService, useClass: PortalDialogService }],
```

Comment:
Please remove provider for `DialogService`

Reply:
done